### DISCUSSION TITLE
[Feature Request] Rebind controls/gestures

### DISCUSSION THREAD

**Discussion by tuglaw** [d4rken-org/capod#373] [Ideas]
Hello,

Would it be possible for CAPod to intercept controls/gestures and rebind them to other functions?

Thank you.

Example (AirPods 4):
- Rebind Single Press in Left Bud to "Volume Down" (previously "Play/Pause")
- Rebind Single Press in Right Bud to "Volume Up" (previously "Play/Pause")
- Rebind Double Press in Left Bud to "Play/Pause" (previously "Play Next")
- etc.

**Comment by kavishdevar** [comment]
afaik, on AirPods some gestures like play pause are hardcoded, and sent from the airpods directly (AVRCP commands), rather than sending button presses signals. But it might be possible for some settings, like mute/end call, if #215 can be fixed.

**Comment by d4rken** [comment]
Maybe possible in a future Android version.
Requires a change in the Android firmware, see #215
